### DISCUSSION TITLE
Flair for trusted users, mods, and staff so users know who they can go to

### DISCUSSION THREAD

**Discussion by ch-stripe** [forem/forem#17766] [Feature Requests]
I asked about this earlier, and was directed here: It would be useful/helpful if we had badges or some kind of flair to show who's:
1) staff (this one is pretty standard)
2) mods (it makes sense readers should know who has powers to moderate which tags, since it tells you they're probably community leaders in that field)
3) trusted users - this one could go either way but it would be great to know who these people are so you can kind of "follow" them the way you might follow other trusted voices on other platforms

**Comment by michael-tharrington** [comment]
Thanks so much for writing this one up Carol! 🙌

I fully agree with ya that flair would be helpful for all of these roles. 

Also, here's [a related issue](https://github.com/forem/forem/issues/9620) that has been lingering around for a while in case the discussion there is helpful with implementation.
